lib/config/main.go

```diff
@@ -24,6 +24,7 @@ type oplogtoredisConfiguration struct {
 	OplogV2ExtractSubfieldChanges bool          `default:"false" envconfig:"OPLOG_V2_EXTRACT_SUBFIELD_CHANGES"`
 	WriteParallelism              int           `default:"1" split_words:"true"`
 	ReadParallelism               int           `default:"1" split_words:"true"`
+	PostgresPersistenceURL        string        `envconfig:"PG_PERSISTENCE_URL"`
```

Comment:
do we need to update the chart?

Reply:
Yup: https://github.com/tulip/tulip/pull/20962